src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleanerWflow.py

```diff
@@ -200,6 +201,7 @@ def docSchema(self):
         docTemplate = [
             ('RequestName', None, (bytes, str)),
             ('RequestType', None, (bytes, str)),
+            ('SubRequestType', None, (bytes, str)),
```

Comment:
Due to this change I want to raise your awareness of consequences of having multiple data-type assignments to a single variable. Why do you do that? Such assignment only creates other problems down in the road making checks, and code to do extra work and hide issue with data-types. All conversions should be done in place and attributes in a document should have single data type. But it is my view and I doubt you'll make the changes. But as reviewer I want to want you about it.

Reply:
Byte and unicode strings was very hard to deal while making WMCore compatible with py2 and py3. As we start removing python2 code and making WMCore python3-only, we should definitely revisit strings and try to converge to a single type as much as we can in the project. Good point!

For now, I am just following the stream here.